Expensify/App
# Show growl when clicking on links in offline mode

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/180868

### Tests / QA
1. Go to settings > about
2. Verify the external links work
3. Go offline
4. Click on an external link
5. Verify that you see a growl
6. Go back online
7. Click on an external link
8. Verify that they work
9. Repeat step 2-8 on Settings > Workspace > (any external link to OldDot on any of these pages like view bills or view invoices)

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/1228807/137003229-5de74c6e-5277-4b6a-be5e-a64689f1fb96.png)


#### Mobile Web
![image](https://user-images.githubusercontent.com/1228807/137003305-0194e715-256b-4926-8460-c9f1b5ad6e8b.png)

#### Desktop
![image](https://user-images.githubusercontent.com/1228807/137001818-c75cf280-2146-43bf-afe9-ec44b7226a18.png)


#### iOS
![image](https://user-images.githubusercontent.com/1228807/137003244-1edb5c49-dbb5-479c-936f-429461e58e6e.png)


#### Android
